src/charts/Dashboard/components/DashboardPlaceholder/DashboardPlaceholder.js

```diff
@@ -40,7 +40,7 @@ const DashboardPlaceholder = (props) => {
           {label}
         </Typography>
       </Grid>
-      {downloadLogsFile && (
+      {downloadLogsFile && scenario.lastRun && (
```

Comment:
Instead of adding a new prop to this component, we could change the file _src/charts/SimplePowerBIReportEmbed/SimplePowerBIReportEmbed.js_ to set `downloadLogsFile` to null when `scenario.lastRun` is null/undefined.
(by the way, if we only need the lastRun, we shouldn't add the whole scenario data as prop :slightly_smiling_face: )